api/v1alpha1/envoyproxy_tracing_types.go

```diff
@@ -111,6 +119,10 @@ type CustomTag struct {
 	// RequestHeader adds value from request header to each span.
 	// It's required when the type is "RequestHeader".
 	RequestHeader *RequestHeaderCustomTag `json:"requestHeader,omitempty"`
+	// Formatter adds value using formatter to each span.
+	// It's required when the type is "Formatter".
+	// +optional
+	Formatter *FormatterCustomTag `json:"formatter,omitempty"`
```

Comment:
any reason we have 2 level here ?
does the similar field for access log use the same field name of `formatter` ?

Reply:
Name does align with the changes from https://github.com/envoyproxy/envoy/pull/41697